lib/routers/profile.js

```diff
@@ -90,6 +90,13 @@ module.exports = () => {
           if (profile.asruUser) {
             throw new BadRequestError('Cannot merge profiles with an ASRU user');
           }
+          return profile;
+        })
+        .then(profile => req.workflow.profileTasks({ profileId: req.profile.id }))
```

Comment:
Per my other comment on https://github.com/UKHomeOffice/asl-service/pull/175

```suggestion
        .then(profile => req.workflow.profileTasks(req.profile.id))
```